### DISCUSSION TITLE
chatbot for wiki. https://polkadot.polkassembly.io/referenda/61

### DISCUSSION THREAD

**Discussion by polkatax** [w3f/polkadot-wiki#5015] [General]
Hello. 

I have a question are regarding a proposal which I submitted (probably prematurely): [https://polkadot.polkassembly.io/referenda/61](https://polkadot.polkassembly.io/referenda/61). 
It's about providing a chatbot for the polkadot wiki. It's yet unclear whether the proposal is accepted. 

The current bot can be accessed [here](https://polkabot.org).

Is it feasable to integrate such a chatbot into the wiki (e.g. via iframe or other means)?

**Comment by profradha** [comment]
Thank you @loanMaster, for the initiative to create a chatbot based on the Polkadot Wiki content. When ChatGPT was announced, we considered adding this to the Wiki, and upon careful evaluation, we decided against it. Here are our reasons -

1. The chatbot technology is still experimental and has yet to prove effective in technical subjects with limited documentation. You can refer to this post on StackExchange https://meta.stackoverflow.com/questions/421831/temporary-policy-generative-ai-e-g-chatgpt-is-banned
2. Polkadot Wiki is under major reorg https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit?pli=1, and the maintainers add 8-10 PRs each week on its content, given that Polkadot is a rapidly evolving protocol https://github.com/paritytech/polkadot/releases 
3. We tested the effectiveness of the chatbots trained on the Wiki content and were unsatisfied with the results. 

We do not plan to integrate a Chatbot in the Wiki anytime soon. Although we do encourage making the Wiki content accessible through emerging technologies like ChatGPT and are happy to get involved in QA testing.
